src/desktop.go

```diff
@@ -183,7 +192,15 @@ func printDesktops(conf *config, desktops []*desktop) {
 		if printSeparator && i > 0 {
 			fmt.Print(dSeparator)
 		}
-		fmt.Printf("[%d] %s", i, v.name)
+		if conf.VerticalSelection {
+			extraIndent := ""
+			if i < 10 {
+				extraIndent = " "
+			}
+			fmt.Printf("%s[%d] %s", extraIndent, i, v.name)
+		} else {
+			fmt.Printf("[%d] %s", i, v.name)
+		}
```

Comment:
Whole block could be simplified to something like this:
```golang
extraIndent := ""
if conf.VerticalSelection && i < 10 {
	extraIndent = " "
}
fmt.Printf("%s[%d] %s", extraIndent, i, v.name)
```

It will help with future maintainability of the code.